pbjson-test/src/lib.rs

```diff
@@ -427,6 +427,34 @@ mod tests {
 
         decoded.string_value = None;
         verify(&decoded, r#"{}"#);
+
+        // Test explicit null optional scalar
```

Comment:
The statefulness makes it a bit hard to read, because what is `decoded` at this point? Would be nice if `decoded` would be specified fully before every `verify[_decode]` block.

Reply:
The line above verifies that decoded is `{}`?